src/Orc.NuGetExplorer.Xaml/Services/Interfaces/IPackageCommandService.cs

```diff
@@ -18,7 +18,11 @@ public interface IPackageCommandService
         Task ExecuteAsync(PackageOperationType operationType, IPackageDetails packageDetails, IRepository sourceRepository = null, bool allowedPrerelease = false);
         Task ExecuteAsync(PackageOperationType operationType, IPackageDetails packageDetails);
         Task<bool> CanExecuteAsync(PackageOperationType operationType, IPackageDetails package);
+        
+        [ObsoleteEx(TreatAsErrorFromVersion = "4.5", RemoveInVersion = "5.0")]
         bool IsRefreshRequired(PackageOperationType operationType);
+        [ObsoleteEx(TreatAsErrorFromVersion = "4.5", RemoveInVersion = "5.0")]
+
```

Comment:
this new line is misleading, is it for "GetPluralActionName"? Please keep them tied together